README.md

```diff
@@ -127,9 +126,14 @@ Sugarizer Server Dashboard is an admin tool for teachers and deployment administ
 * Graphic and request: display graphics and report on previous data.
 
 To login to the Dashboard the first time, you will have to create an admin account using this command:
-
+        
 	sh add-admin.sh admin password http://127.0.0.1:8080/auth/signup
 
+Incase using Docker to login the Dashboard for the first time 
+
+	docker exec -it <id of sugarizer-server_server> bash

```

Comment:
We can also consider using `docker-compose exec server bash` instead of this command. That way we don't have to worry about the ID of the container. @nitrophenol can you test that?

@llaske what is your opinion on using `docker-compose exec` instead?